index.html

```diff
@@ -169,6 +169,19 @@ <h2>Getting Started</h2>
       <a class="top" href="#">▲</a>
     </section>
 
+    <section id="further-reading">
+      <h2>Further Reading</h2>
+      <p>These are some extra resources to get you started with specific use-cases of Rust IPFS:</p>
+      <div>
+        <ul>
+          <li><a href="https://rs-ipfs.github.io/offchain-ipfs-manual/">The offchain::ipfs manual</a>: a project integrating IPFS and Substrate.</li>
+          <!-- TODO: Add link once merged -->
```

Comment:
Remove this comment and we'll me good to merge!

Reply:
Fixed 👍